debian/patches/0069-add-tonemap-videotoolbox.patch

```diff
@@ -496,7 +508,15 @@ new file mode 100644
 +        *g4 *= factor_g;
 +        *b4 *= factor_b;
 +    } else {
-+        float4 sig = fmax(fmax(*r4, fmax(*g4, *b4)), FLOAT_EPS);
++        float4 sig;
++        if (is_tone_mode_max) {
++            sig = fmax(fmax3(*r4, *g4, *b4), FLOAT_EPS);
++        } else {
++            sig = fmax((*r4 * .2126f + *g4 * .7152f + *b4 * .0722f), FLOAT_EPS);
```

Comment:
Shouldn't the [BT.2020/BT.2100 primaries](https://en.wikipedia.org/wiki/Rec._2100#Luma_coefficients) `(0.2627f; 0.678f; 0.0593f)` be used to calculate relative luminance? It happens before the RGB values ​​are tonemapped. I tested it and it works as expected too.

Reply:
Probably we should. I use the sRGB primaries because that was what Apple uses in the HDR image processing sample code, but their colorspace could be different which still uses sRGB primaries but extended to HDR spaces.